src/gc-cpu-runner/gc-cpu-runner.cpp

```diff
@@ -38,6 +39,7 @@ int main(int argc, char **argv) {
   llvm::InitializeNativeTargetAsmParser();
 
   mlir::DialectRegistry registry;
+  mlir::registerAllDialects(registry);
```

Comment:
I would suggest to find out which dialect we really need, instead of all dialects initialization.

Reply:
Only `arith` is needed. I narrowed down the scope.